.gitattributes

```diff
@@ -1,3 +1,3 @@
 docs/*.md linguist-generated=true
 pnpm-lock.yaml linguist-generated=true
-e2e/node_package/rules_foo/npm_repositories.bzl linguist-generated=true
+e2e/link_node_package/rules_foo/npm_repositories.bzl linguist-generated=true
```

Comment:
wonder if we should use a filename convention for vendored repositories.bzl files